### PR TITLE
Use Laravel ResponseHeaderBag to send Set-Cookie Header instead of Swoole function

### PR DESCRIPTION
**Pull request by NiroDeveloper**
At the moment the SwooleClient uses the cookie or rawCookie function of the SwooleResponse to send cookies.
With these functions it is not possible to send the Max-Age cookie parameter, only an absolute expires time is sent.
Which leads to the problem that clients without a valid time do not save the cookie correctly.

To fix this and simplify the code, this pull request removes the swoole cookie function and reuses the cookie header function from the Laravel ResponseHeaderBag class.

This has the further advantage that the responses between a normal Laravel PHP-FPM server and Laravel Octane do not differ.

on-behalf-of: @e-solutions-GmbH <info@esolutions.de>

### CODE REVIEW

File: src/Swoole/SwooleClient.php [hunk 1]
```diff
@@ -171,8 +171,9 @@ public function respond(RequestContext $context, OctaneResponse $octaneResponse)
     /**
      * Send the headers from the Illuminate response to the Swoole response.
      *
-     * @param  \Symfony\Component\HttpFoundation\Response  $response
-     * @param  \Swoole\Http\Response  $response
+     * @param   \Symfony\Component\HttpFoundation\Response  $response
+     * @param   \Swoole\Http\Response                       $swooleResponse
+     *
```
Comment (driesvints) [inline src/Swoole/SwooleClient.php:176]: Please undo these changes.

Comment (NiroDeveloper) [reply src/Swoole/SwooleClient.php]: Reverted now ... sorry, problems with IDE formatting

**Comment by taylorotwell** [comment]
Eh, I'm a little uncomfortable just deleting this code. It was put there for a reason. We would need better explanation and more confidence from a Swoole maintainer that just deleting this code is the proper answer.

**Comment by NathanFreeman** [comment]
@NiroDeveloper @taylorotwell @driesvints Hi.
I have made a pr about this problem and maybe it would be helpful.
https://github.com/swoole/swoole-src/pull/5034
https://github.com/swoole/swoole-src/pull/5035
